### PR TITLE
TA-3592 added file uploader component

### PR DESCRIPTION
**Pull request by averysmithproductions**
for jira ticket https://us-sba.atlassian.net/browse/TA-3592

AC:

Newly designed form needs to support current paradigm of:
 - [x] Multiple file uploads

to demo, please visit: http://avery.ussba.io/filetransfer

### CODE REVIEW

File: src/client/components/pages/file-transfer-service-page/file-transfer-service-page.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          <TextArea
+            className={styles.message}
+            id="message"
+            label="Mesage"
+            onChange={ e => this.setState({ textAreaValue: e }) }
+          />
+          <h2>Attach files</h2>
+          <div data-testid="file-uploader">
+            <FileUploader label="File upload area" onChange={ files => this.setState({ filesValue: files }) }/>
+          </div>
```
Comment (averysmithproductions) [inline src/client/components/pages/file-transfer-service-page/file-transfer-service-page.jsx:56]: The FileUploader Component is located here.

File: src/client/components/pages/file-transfer-service-page/file-transfer-service-page.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          />
+          <TextArea
+            className={styles.message}
+            id="message"
+            label="Mesage"
+            onChange={ e => this.setState({ textAreaValue: e }) }
+          />
+          <h2>Attach files</h2>
+          <div data-testid="file-uploader">
+            <FileUploader label="File upload area" onChange={ files => this.setState({ filesValue: files }) }/>
```
Comment (amyd11) [inline src/client/components/pages/file-transfer-service-page/file-transfer-service-page.jsx:55]: Nice this looks like the callback function will give this component access to the files

File: src/client/components/atoms/file-uploader/file-uploader.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		this.setState({ files: acceptedFiles, list }, () => {
+			onChange({ files: acceptedFiles })
+		})
+	}
+	render() {
+		const { list } = this.state
+		const { label } = this.props
+		return (
+			<div className={styles.fileUploader}>
+	 			<label for="dropzone-files" className={styles.label}>{label}</label>
```
Comment (amyd11) [inline src/client/components/atoms/file-uploader/file-uploader.jsx:39]: It doesn't seem like the screen reader is picking up on the label when tabbing through the form. Do you think we need a tabIndex or something?

File: test/jest/client/components/pages/file-transfer-service-page.jsx/file-transfer-service-page.test.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import sinon from 'sinon'
+import FileTransferServicePage from 'pages/file-transfer-service-page/file-transfer-service-page.jsx'
+
+describe('FileTransferServicePage', () => {
+	it('should have one FileUploader component', () => {
+		const component = shallow(<FileTransferServicePage />)
+		expect(component.find('[data-testid="file-uploader"]').length).toEqual(1)
+	})
```
Comment (kscoulter) [inline test/jest/client/components/pages/file-transfer-service-page.jsx/file-transfer-service-page.test.jsx:10]: you can also use `.toHaveLength(1)`

**Pull request merged**